src/main/java/zd/zero/waifu/motivator/plugin/integrations/ErrorReporter.kt

```diff
@@ -136,4 +135,47 @@ class ErrorReporter : ErrorReportSubmitter() {
         if (edition != null) appName += " ($edition)"
         return Pair(appInfo, appName)
     }
+
+    private fun getMinifiedConfig(): String {
```

Comment:
![Wow!](https://media.giphy.com/media/qVVVfmHDMBZug/giphy.gif)

:laughing: This is fine, there really isn't a clean solution to this problem. In the end, reflection is being used (just indirectly in this case)